packages/components/src/select-control/style.scss

```diff
@@ -74,6 +74,13 @@
 			font-size: 12px;
 			margin-top: -$gap-small;
 		}
+
+		&.is-disabled {
```

Comment:
Could this use the `:disabled` pseudo selector instead of adding a class?

Reply:
Thank you @jeffstieler for your review,

I'm not sure if I understood your suggestion, sorry if I missed something. The class `is-disabled` is adding styles to a few elements (not only to an `input`) so I'm not sure if we should use an attribute in the parent div to achieve that.

I think that the `:disabled` pseudo-class won't work here. It should if we use an attribute selector (square brackets) to apply the styles though.

On the other hand, as far as I could see, when we want to "disable" something different than a `button` or an `input` we use a class (like [here](https://github.com/woocommerce/woocommerce-admin/blob/main/client/profile-wizard/steps/business-details/flows/selective-bundle/style.scss#L4) and [here](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/src/text-control-with-affixes/style.scss#L48)) rather than an attribute addition to the element (I couldn't find any examples). 

If you still think that I should continue with the disabled attribute in the div, please let me know and I'll change it.

Reply:
I should have slowed down and read more carefully, this is fine as-is. 😅